### PR TITLE
Provide a correct path for saving model

### PR DESCRIPTION
**Pull request by MilenaOlech**

### CODE REVIEW

File: models/model.py [hunk 1]
```diff
@@ -17,7 +18,7 @@ def __init__(self,
                  num_classes=1000,
                  feature_extractor_name="cnn_extractor",
                  transformer_name="vanilla_transformer",
-                 model_save_dir=""):
+                 model_save_dir= "/" + os.getcwd() + "/models/__pycache__/model.cpython-36.pyc"):
```
Comment (wiktorfilipiuk) [inline models/model.py:21]: Can we make paths OS agnostic? :)

Comment (MilenaOlech) [reply models/model.py:21]: Probably we can, but could you explain why it's a problem?

**Comment by aga0** [comment]
In the future it would be useful to not to have this argument hardcoded in this script and pass the path as a variable from the previous step, but having  a functional pipeline is more important now, so let's have it this way.

**Review by aga0 — APPROVED** [review]

File: models/model.py (hunk 1 above)
Comment (majsylw) [reply models/model.py]: Use rather os.path.join() method than "/" or "\" chars which depend on system

**Review by AgaMiko — APPROVED** [review]

**Pull request closed without merging**